src/views/Main/Catalog/Components/EntitySet.js

```diff
@@ -120,15 +120,15 @@ export class EntitySet extends React.Component {
     let downloadOptions;
     let requestOptions;
     const permissions = this.props.permissions;
-    if (permissions.includes(Consts.WRITE.toUpperCase())) {
+    if (permissions.includes(Consts.WRITE)) {
```

Comment:
got it. thought it was a string. didn't realize `.includes()` is available to arrays also